pyproject.toml

```diff
@@ -73,6 +73,11 @@ jax = { version = "0.4.25", extras = [
   "cpu",
 ], source = "jaxsource", optional = true }
 
+# design
```

Comment:
I didnt have these installed and the error was nicely handled (only errored when the appropriate method was called)